### DISCUSSION TITLE
[Nuxt 3] 사이드 프로젝트 만들기 - 개발 환경 설정편 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#21] [Blog Comments]
# [Nuxt 3] 사이드 프로젝트 만들기 - 개발 환경 설정편 - peterkimzz

저번 사이드 프로젝트 만들기 - 기획편의 다음 편입니다. 이번엔 nuxt3의 주요 변경사항 일부를 알아보고, 쾌적한 개발 환경을 위해 몇 가지 세팅을 해보도록 하겠습니다.
일단 새로운 Nuxt3 프로젝트를 생성합시다. 터미널을 열고 아래 명령어를 입력합시다.
저는 웹 

https://www.peterkimzz.com/nuxt3-sideproject-2/

**Comment by suzy-thetraveler** [comment]
도커 구글링하다가 들어오게 되었는데 정말 좋은 정보들이 많이 있어서 보물을 발견한 기분이예요! 좋은 정리 감사합니다!!

**Reply by peterkimzz** [reply]
@suzy-thetraveler 감사합니다! 근데 요새 게을러져서 글을 못올리고 있네요 😂

**Comment by zealot71** [comment]
tailwind 설치하는 과정에서 nuxt.config.ts 파일에서 Cannot start nuxt:  Cannot find module 'nuxt3'  에러가 뜨네요...
어떤 것 때문에 그럴까요...

**Reply by peterkimzz** [reply]
아, nuxt3 프로젝트에서 `tailwindcss` 를 설치할 때 문제가 되신다면 이렇게 한 번 바꿔보세요. 개선이 되어서 훨씬 쉽게 설치가 가능해졌습니다.

```bash
yarn add --dev @nuxtjs/tailwindcss
```

```ts
// nuxt.config.ts
export default defineNuxtConfig({
    modules: ['@nuxtjs/tailwindcss'],
    // 나머지 옵션은 전부 제거해도 됩니다.
})
```

자세한 설치 과정은 https://tailwindcss.nuxtjs.org/getting-started/setup 을 참고해주세요.

이 설치 과정을 블로그 글에 업데이트 해두겠습니다. 감사합니다!

**Comment by flaghrbs1004** [comment]
nuxt3 를 공부하면서 찾아온 사이트에서 많은 인사이트가 담겨있어서 들어온것에 감사합니다.
useFetch를 사용해서 백엔드서버 url를 baseURL로 설정해주기 위해서 검색하다가 들어왔습니다
'~/composables/useApi.ts' 파일을 생성 후 보여주신 코드를 넣었습니다.
자동으로 import 해준뒤에  따로 해야하는 작업이 있을까요?
저는 WebStorm을 사용중인데 'No usages found in All Places' 사용되는 곳이 없다고 나와요..

**Reply by flaghrbs1004** [reply]
nuxt devtools 에서 imports 부분을 확인해보니 User composables 부분에서 확인이 가능하네요 './composables/useApi.ts'  하지만 Not used 필터에 보여지고 있어요

**Reply by peterkimzz** [reply]
안녕하세요. 음 WebStorm에서 타입스크립트 인식이 잘 안된게 아닐까 하는데 혹시 그 부분을 먼저 확인해보시겠어요?
